example/example.py

```diff
@@ -20,4 +20,4 @@
 errors = schema.validate(test_data)
 
 for error in errors:
-    print(error)
+    print(error)
```

Comment:
This

Reply:
sorry, what?

Reply:
I believe you removed the linebreak at the end of the file